packages/react-core/src/components/Page/Page.tsx

```diff
@@ -197,6 +210,36 @@ export class Page extends React.Component<PageProps, PageState> {
       isNavOpen: mobileView ? mobileIsNavOpen : desktopIsNavOpen
     };
 
+    const nav = tertiaryNav ? (
+      isTertiaryNavWidthLimited ? (
+        <div className={css(styles.pageMainNav, styles.modifiers.limitWidth)}>
+          <div className={css(styles.pageMainBody)}>{tertiaryNav}</div>
+        </div>
+      ) : (
+        <div className={css(styles.pageMainNav)}>{tertiaryNav}</div>
+      )
+    ) : null;
+
+    const crumb = breadcrumb ? (
+      isBreadcrumbWidthLimited ? (
+        <section className={css(styles.pageMainBreadcrumb, styles.modifiers.limitWidth)}>
+          <div className={css(styles.pageMainBody)}>{breadcrumb}</div>
+        </section>
+      ) : (
+        <section className={css(styles.pageMainBreadcrumb)}>{breadcrumb}</section>
+      )
+    ) : null;
+
+    const isGrouped = isTertiaryNavGrouped || isBreadcrumbGrouped;
```

Comment:
oh, i think you'll also want to add `|| additionalGroupedContent` in case they don't want to group tertiary and breadcrumb but still show some grouped sticky content at the top?

Reply:
Will add. Because users may also use the group component directly, they could also do that instead of using these props in this case.